### PR TITLE
CLI Docs Update updated instructions for file with windows.tar.gz extension, describe appsody list and init changes for multiple repositories, include latest cli-commands update doc.

### LINKED ISSUE #152: Upade install instructions for windows.tar.gz
Do not integrate this issue until the corresponding issue is delt with in appsody/appsody and the 0.3.0 release is ready.

The following will need to change at the next CLI release boundary:

Download the Appsody binaries for Windows from the Appsody releases page into the directory. The file is named appsody-v.r.m-windows.tar, where v.r.m indicates the release tag.
Extract the files by running tar -xvf appsody-v.r.m-windows.tar.
To install Appsody, run the following setup command:

TO

Download the Appsody binaries for Windows from the Appsody releases page into the directory. The file is named appsody-v.r.m-windows.tar.gz, where v.r.m indicates the release tag.
Extract the files by running tar -xvf appsody-v.r.m-windows.tar.gz
To install Appsody, run the following setup command:

### LINKED ISSUE #153: Update initializeproject.md to reflect multiple repos and v2 changes to appsody init and appsody list
appsody init now can take the syntax

appsody init <repo-name>/stack

appsody list takes the parameter <repo-name> and displays the <repo-name> column in its output.

### PR DESCRIPTION
**Pull request by kewegner**
Fixes #152 and fixes #153
Added instructions related to the .gz extension for the windows file: appsody-v.r.m-windows.tar.gz for install and update.

### CODE REVIEW

File: content/docs/getting-started/installation.md [hunk 1]
```diff
@@ -61,8 +61,11 @@ Note: Appsody runs only on Windows 10.
 Follow these steps:
 
 1. Create a directory for Appsody on your Windows 10 system.
-2. Download the **Appsody binaries for Windows** from the [Appsody releases page](https://github.com/appsody/appsody/releases) into the directory. The file is named `appsody-v.r.m-windows.tar`, where `v.r.m` indicates the release tag.
-3. Extract the files by running `tar -xvf appsody-v.r.m-windows.tar`.
+2. Download the **Appsody binaries for Windows** from the [Appsody releases page](https://github.com/appsody/appsody/releases) into the directory. The file is named `appsody-v.r.m-windows.tar.gz`, where `v.r.m` indicates the release tag.
+3. Extract the files by running `tar -xvf appsody-v.r.m-windows.tar.gz`.
+
+Note: If you choose to extract the files using a GUI tool you may have to extract the `appsody-v.r.m-windows.tar.gz` file and extract the extracted `appsody-v.r.m-windows.tar.gz` file afterwords.
```
Comment (tnixa) [inline content/docs/getting-started/installation.md]: should be "...the extracted `appsody-v.r.m-windows.tar` file afterwords"

File: content/docs/getting-started/installation.md [hunk 2]
```diff
@@ -142,9 +145,11 @@ sudo yum update appsody-v.r.m-1.x86_64.rpm -y
 
 ## Upgrading on Windows
 To upgrade Appsody on your system, follow these steps: 
-1) Download the **Appsody binaries for Windows** from  [Appsody releases page](https://github.com/appsody/appsody/releases). The file is named `appsody-v.r.m-windows.tar`, where `v.r.m` indicates the release tag.
+1) Download the **Appsody binaries for Windows** from  [Appsody releases page](https://github.com/appsody/appsody/releases). The file is named `appsody-v.r.m-windows.tar.gz`, where `v.r.m` indicates the release tag.
 2) Move the file to the directory where you stored the existing Appsody binaries.
-3) Untar the file there: `tar -xvf appsody-v.r.m-windows.tar`.
+3) Untar the file there: `tar -xvf appsody-v.r.m-windows.tar.gz`.
+
+Note: If you choose to extract the files using a GUI tool you may have to extract the `appsody-v.r.m-windows.tar.gz` file and extract the extracted `appsody-v.r.m-windows.tar.gz` file afterwords.
```
Comment (tnixa) [inline content/docs/getting-started/installation.md]: should be "...the extracted `appsody-v.r.m-windows.tar` file afterwords"

File: content/docs/using-appsody/initializing-project.md [hunk 3]
```diff
@@ -23,15 +23,31 @@ Here is an example of the output produced by the `appsody list` command:
...
+appsodyhub  	java-spring-boot2	0.3.2  	Spring Boot using OpenJ9 and Maven           
+appsodyhub  	nodejs-express   	0.2.2  	Express web framework for Node.js            
+appsodyhub  	nodejs           	0.2.2  	Runtime for Node.js applications             
+appsodyhub  	swift            	0.1.0  	Runtime for Swift applications               
+
+
 ```
 
-When you have chosen your stack, run `appsody init <stack>` for the stack of your choice. This command downloads the default project template for the stack and initializes Appsody for local development.
+When you have chosen your stack, run `appsody init <repository-name>/<stack>` for the repository and stack of your choice. This command downloads the default project template for the stack and initializes Appsody for local development.  If you elect to only specify the stack, e.g. `appsody init <stacK>`, the default repository is assumed.
```
Comment (tomleah) [inline content/docs/using-appsody/initializing-project.md]: Lowercase "K"
`appsody init <stacK>` -> `appsody init <stack>`

**Comment by tomleah** [comment]
Pinning to #152 in zenhub for convenience.

**Comment by tomleah** [comment]
@tnixa are your comments resolved? Could you approve this if so

**Comment by kewegner** [comment]
title has been added back in to cli-comands.md

**Pull request merged**